monailabel/scribbles/infer.py

```diff
@@ -16,30 +16,21 @@
     AddBackgroundScribblesFromROId,
     ApplyGraphCutOptimisationd,
     MakeISegUnaryd,
+    MakeLikelihoodFromScribblesGMMd,
     MakeLikelihoodFromScribblesHistogramd,
 )
 from monailabel.transform.post import BoundingBoxd, Restored
 
 
-class HistogramBasedGraphCut(InferTask):
+class MyScribblesLikelihoodInferTask(InferTask):
```

Comment:
can we name it better? Instead of Myxxx

Reply:
Sure, how about `ScribblesLikelihoodInferTask`?

Reply:
Changed it to `ScribblesLikelihoodInferTask`